integration-tests/gradle/projects/it-multiplatform-0/build.gradle.kts

```diff
@@ -25,7 +25,10 @@ kotlin {
         }
         named("commonMain") {
             dependencies {
-                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.9")
+                if (properties["dokka_it_kotlin_version"] in listOf("1.4.32", "1.5.31"))
+                    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.9")
+                else
+                    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4")
```

Comment:
Could you leave a comment explaining why this is needed?

Reply:
Also, see https://github.com/Kotlin/kotlinx.coroutines/blob/81e17dd37003a7105e542eb725f51ee0dc353354/gradle.properties#L8